code/datums/repositories/client.dm

```diff
@@ -0,0 +1,61 @@
+var/const/NO_CLIENT_CKEY = "*no ckey*"
+
+var/global/repository/client/client_repository = new()
```

Comment:
What is this for?

Reply:
Psure it's a global list of lite clients or whatever.